### PR TITLE
Add selected reporting id to response

### PR DESCRIPTION
**Pull request by qingxinwu**
add selectedBuyerANdSellerReportingId

### CODE REVIEW

File: draft-ietf-bidding-and-auction-services.md [hunk 1]
```diff
@@ -778,6 +778,11 @@ response = {
   ; If not present, map as Null.
   ? buyerAndSellerReportingId: tstr,
 
+  ; Optional SelectedBuyerAndSellerReportingId of the winning Ad
+  ; Maps directly to https://wicg.github.io/turtledove/#server-auction-response-buyer-and-seller-reporting-id. (TODO: update before merge)
```
Comment (qingxinwu) [inline draft-ietf-bidding-and-auction-services.md:782]: update after W3C spec's [pull request](https://github.com/WICG/turtledove/pull/1348) that defines the new field is merged.

**Review by brusshamilton — APPROVED** [review]
LGTM modulo updating the link referenced in your open comment.

File: draft-ietf-bidding-and-auction-services.md (hunk 1 above)
Comment (qingxinwu) [reply draft-ietf-bidding-and-auction-services.md]: done.

**Comment by qingxinwu** [comment]
> LGTM modulo updating the link referenced in your open comment.

done.

**Review by brusshamilton — APPROVED** [review]

**Pull request merged**